README.md

```diff
@@ -1,90 +1,99 @@
-3. 🏠 Engineers get a reproducible, composable, and portable environment for all their testing and development needs
+- A language for declaring distributed application in Python syntax ([Starlark](https://github.com/google/starlark-go/blob/master/doc/spec.md))
+- A packaging system for sharing and reusing distributed application components
+- A runtime that makes a Kurtosis app Just Work, independent of whether it's running on Docker or Kubernetes, local or in the cloud
+- A set of tools to ease common distributed app development needs (e.g. a log aggregator to ease log-diving, automatic port-forwarding to ease connectivity, a `kurtosis service shell` command to ease container filesystem exploration, etc.)
 
 Why should I use Kurtosis?
 ==========================
-Kurtosis is valuable to blockchain infrastructure developers because, when compared to alternatives, our _environment definitions_ are:
-* 🧱 **Modular** - parts of the system can be combined or be taken apart at the container level
-* ✏️ **Easy to write** - we know `.yaml` doesn't cut it, so we use a Python dialect called Starlark, written and used by Google ([Bazel](https://bazel.build/)) and Meta ([Buck](https://buck2.build/))
-* 🪜 **Sequential** - catering to the specific node-bootstrapping needs of blockchains
-* 📈 **Scale-able** - operate at any scale you need, limited only by the underlying hardware
-* 🔌 **Integratable** - plug & play integration with your existing infrastructure & made to be built on top of
-* ♻️ **Reproducable** - Completely deterministic and idempotent: given the same inputs, the system will start up the exact same way, every time
-* 🆓 **Open source** - Kurtosis is transparent and will be free, forever
-
-Still curious?
-==============
-* Kurtosis Web3 use cases [here](https://web3.kurtosis.com).
-* Learn about "why Kurtosis?" [here](https://docs.kurtosis.com/#why-use-kurtosis).
-* To read about the architecture, go [here](https://docs.kurtosis.com/explanations/architecture).
-* To get a peek at where we're going with the product, check out the roadmap [here](https://github.com/kurtosis-tech/kurtosis/wiki/Short%E2%80%90term-Roadmap).
-* For real examples, go [here](https://github.com/kurtosis-tech/awesome-kurtosis)
-
-Kurtosis Kloud Early Access
-===========================
+Kurtosis shines when creating, working with, and destroying self-contained distributed application environments. Currently, our users report this to be most useful when:
 
-If you're looking to run Kurtosis environments on the cloud, look no further! 
-We're excited to launch an early access offering for Kurtosis Kloud. Once you [sign up](https://mp2k8nqxxgj.typeform.com/to/U1HcXT1H), we'll reach out to you with the next steps.
+- You're developing on your application and you need to rapidly iterate on it
+- You want to try someone's containerized service or distributed application on your local machine
```

Comment:
```suggestion
- You want to try someone's containerized service or distributed application quickly and without the overhead of learning how to set up the environment, dependencies, and what arguments to use.
```
Re-wording based on what I believe is the reason why our users pick Kurtosis to spin up their systems over a myriad of other tools. Also, I don't believe the "local machine" part of this is relevant - we've already told folks it works remotely or locally, over Docker or k8s. Let's keep the backend part vague because it truly is independent of Kurtosis.

Reply:
resolved over Slack